appeals/web/package.json

```diff
@@ -13,7 +13,7 @@
     "clean": "rimraf .turbo node_modules dist",
     "clean:dev": "rimraf dist src/server/static/scripts src/server/static/styles",
     "dev": "cross-env NODE_ENV=local npm-run-all --silent clean:dev sass rollup --parallel --race watch:** nodemon",
-    "lint:js": "npx eslint .",
+		"lint:js": "npx eslint .",
```

Comment:
Not a problem as this won't break anything, but was this indentation added by accident?